### DISCUSSION TITLE
Configuring NoisePy for fiber data (non-seismological)

### DISCUSSION THREAD

**Discussion by zibiwilczynski** [mdenolle/NoisePy#316] [Q&A]
Hi, 

After reading your paper (Jiang and Denolle 2020, SRL), I am interested in using NoisePy for passive data processing of fibre DAS data surface wave dispersion analysis. It seems your package would suit my needs, and I appreciate your effort in making it available.

However, I cannot correctly configure NoisePy to read and work on my data. I created an ASDF (.h5) and added waveform and StationXML for every station (with some problems). I am then trying to reproduce the given tutorials, modifying the code to read from my file. No success though.

May I ask for help on how to correctly write an ASDF file so that it is readable by NoisePy, and what NoisePy functions to use to read it?

I would appreciate your help
Zibi

**Comment by niyiyu** [comment]
Transferred to Issue #326

**Comment by mdenolle** [comment]
@ zibiwilczynski, @niyiyu transferred the discussion elsewhere.
We do not recommend either ASDF or StationXML for DAS data and metadata at all. We have explored H5 DAS data store and cloud-native formats. Please join the discussion instead of this as an issue.

**Reply by ZHONG-YICHEN** [reply]
Hi! After reading the tutorial, I successfully used the DASH5DataStore function to convert my DAS data into the raw_store required by noisepy. However, during the subsequent cross-correlation process, I found that the program seemed to crash due to station information issues, as shown in the figure. Could you please advise on how to resolve this issue?
<img width="1431" height="812" alt="image" src="https://github.com/user-attachments/assets/ababd702-e925-48ac-a990-a2e1d6938e81" />

**Reply by niyiyu** [reply]
Hi @ZHONG-YICHEN, 

Could you provide some debugging information so that we can better pinpoint where the problem is?

In the mean time, please raise the issue [here](https://github.com/noisepy/NoisePy/issues) so that we can better track it.

Best,
Yiyu

**Reply by ZHONG-YICHEN** [reply]
Sure, thanks for your help!
